styles/style.css

```diff
@@ -5,246 +5,310 @@
+ color: var(--grey-dark);
+ font-size: 1rem;
 }
 
 h1,h2,h3,h4,h5,h6 {
-  margin-top: 0;
-  margin-bottom: 0;
+ margin-top: 0;
+ margin-bottom: 0;
 }
 
 p {
-  margin-top: 0;
-  margin-bottom: 0;
+ margin-top: 0;
+ margin-bottom: 0;
 }
 
-.header,
+header,
 .content,
-.footer {
-  padding: 0 1.5rem;
-  max-width: 1200px;
-  margin: 0 auto;
+footer {
+ padding: 0 1.5rem;
+ max-width: 1200px;
+ margin: 0 auto;
 }
 
 /* Header */
 
-.header {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  height: 5rem;
-  margin-bottom: 1rem;
-  padding-top: 1rem;
+header {
+ display: flex;
+ align-items: center;
+ justify-content: space-between;
+ height: 5rem;
+ margin-bottom: 1rem;
+ padding-top: 1rem;
 }
 
 .header__logo {
-  max-height: 90px;
-  width: auto;
+ max-height: 90px;
+ width: auto;
 }
 
 /* Navigation */
 
 .navigation__list {
-  display: flex;
-  list-style: none;
+ display: flex;
+ list-style: none;
 }
 
 .navigation__item {
-  padding: 0.5rem 0;
+ padding: 0.5rem 15px;
 }
 
 .navigation__link {
-  color: var(--grey-dark);
-  font-weight: 600;
-  text-transform: uppercase;
-  text-decoration: none;
+ color: var(--grey-dark);
+ font-weight: 650;
+ text-transform: uppercase;
+ text-decoration: none;
+}
+
+.navigation__link:hover{
+ color: var(--orange-dark);
 }
 
 
 /* Text styles */
 
 .text-highlight {
-  color: var(--orange-dark);
-  font-weight: 600;
+ color: var(--orange-dark);
+ font-weight: 650;
 }
 
 .arrow-right {
-  display: inline-block;
-  height: 0;
-  width: 0;
-  border: 6px solid var(--white);
-  border-left-color: var(--orange-dark);
+ display: inline-block;
+ height: 0;
+ width: 0;
+ border: 6px solid var(--white);
+ border-left-color: var(--orange-dark);
 }
 
 
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+ /* INSERT BUTTON STYLES HERE */
+
 
+.firstBS{
+ background-color: var(--orange-dark);
+ color: var(--white);
+ padding: 5px 10px;
+ border: none;
+ border-radius: 5px;
+}
+.firstBS:hover{
+ background-color: var(--grey-dark);
+}
+
+.secondBS{
+ background-color: var(--white);
+ color: var(--orange-dark);
+ padding: 5px 10px;
+ border: none;
+ border-radius: 5px;
+}
+.secondBS:hover{
+ background-color: var(--orange-light);
+}
 
 /* Content */
 
 .content {
-  display: flex;
-  flex-direction: column;
+ display: flex;
+ flex-direction: column;
 }
 
 /* Alert */
 
 .alert {
-  background-color: var(--orange-light);
-  border-left: 4px solid var(--orange-dark);
-  border-radius: 4px;
-  font-size: 0.85rem;
-  line-height: 1.3;
-  padding: 1.5rem 1rem;
-  margin-bottom: 2rem;
+ background-color: var(--orange-light);
+ border-left: 4px solid var(--orange-dark);
+ border-radius: 4px;
+ font-size: 0.9rem;
+ line-height: 1.3;
+ padding: 1.5rem 1rem;
+ margin-bottom: 2rem;
 }
 
+
 /* hero */
 
 .hero {
-  background-image: url("header-bike.jpg");
-  background-color: var(--grey-light);
+ background-image: url("/images/header-bike.jpg");
```

Comment:
in order to retrieve the background image, try to start from the location of the file you are working on:
```
styles/style.css
```
and how from there you would get to the file you need:

```
images/header-bike.jpg
```
In this case you would need to go a level up and then retrieve the image.

https://www.ibm.com/docs/en/zos/2.1.0?topic=names-dot-notation

https://www.loom.com/share/69f1549e89a740b8864c11c2b85a5d66